var/spack/repos/builtin/packages/python/package.py

```diff
@@ -945,3 +945,17 @@ def remove_files_from_view(self, view, merge_map):
                 view.remove_file(src, dst)
             else:
                 os.remove(dst)
+
+    def test(self):
+        # guaranteed executable name
```

Comment:
I'm not sure how guaranteed this is. For example, if someone is using an external Python installation on Blue Waters, they tend to bundle `python3.4`, `python3.5`, and `python3.6` all in the same directory.

Reply:
Ok, changed it to use `self.command.name`.